general/git.md

```diff
@@ -0,0 +1,34 @@
+# Git & GitHub
+For source control we use GitHub. All projects are repositories under [scrambledeggs](https://github.com/scrambledeggs).
+
+## Workflow
+- Prefer to adapt [Gitlab flow](https://about.gitlab.com/2014/09/29/gitlab-flow/)
+- The canonical version of the code for active development lives in `master`
+- When a new feature is worked on, a branch will be created from `master`
+- Use descriptive feature branch names and name them using [`kebab-case`](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles)
+- For fixes, treat the same as features but add `fix` to branch name i.e `fix-your-face`
```

Comment:
What about for branches like `release`? Or do we avoid making a branch for release preparations, database packaging etc?